README.md

```diff
@@ -10,8 +10,7 @@
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `pluggy_elixir` to your list of dependencies in `mix.exs`:
+Add `pluggy_elixir` to your list of dependencies in `mix.exs`:
```

Comment:
What do you think about add an explicit example to turn copy/paste easier?
```suggestion
Add `{:pluggy_elixir, "~> 0.1"}` to your list of dependencies in `mix.exs`:
```

Reply:
There is a snippet bellow this line.